### PR TITLE
Add visulization->visualization

### PR DESCRIPTION
**Pull request by tkoyama010**
See https://github.com/pyvista/pyvista-tutorial/pull/86

### CODE REVIEW

**Review by luzpaz — APPROVED** [review]

**Review by peternewman — APPROVED** [review]
Care to add the visuali**s**ation variants for the UK too? LGTM

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -35804,6 +35804,7 @@ visuallization->visualization
 visualy->visually
 visualyse->visualise, visualize,
 visualzation->visualization
+visulization->visualization
```
Comment (tkoyama010) [inline codespell_lib/data/dictionary.txt:35807]:
```suggestion
visualzation->visualisation, visualization
visulization->visualisation, visualization
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -35803,7 +35803,8 @@ visuallisation->visualisation
 visuallization->visualization
 visualy->visually
 visualyse->visualise, visualize,
-visualzation->visualization
+visualzation->visualisation, visualization
+visulization->visualisation, visualization
```
Comment (tkoyama010) [inline codespell_lib/data/dictionary.txt:35807]:
```suggestion
visualzation->visualisation, visualization,
visulization->visualisation, visualization,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -35803,7 +35803,8 @@ visuallisation->visualisation
 visuallization->visualization
 visualy->visually
 visualyse->visualise, visualize,
-visualzation->visualization
+visualzation->visualisation, visualization,
+visulization->visualisation, visualization,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:35807]: Please add plural as well, TIA

Comment (tkoyama010) [reply codespell_lib/data/dictionary.txt:35807]:
```suggestion
visulization->visualisation, visualization,
visulizations->visualisations, visualizations,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -35803,7 +35803,9 @@ visuallisation->visualisation
...
+visulizations->visualisations, visualizations,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:35808]: Sorry, I meant the equivalent typo with an s, e.g. `visualsation->visualisation`.

It's unlikely if someone typed ...z... that they actually meant ...s... (unless they didn't know which one to use, in which case offering both will still leave them confused. But e.g. visualiation could usefully offer both variants.

Also you can edit your own files directly if you want without bothering with suggestions, go to Files changed, then ... menu on a file, then Edit file.

**Review by peternewman — CHANGES_REQUESTED** [review]
Sorry, can you add the British English variants in a different way.

Remember to put them in the right place or sort the dictionary too.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (tkoyama010) [reply codespell_lib/data/dictionary.txt:35808]: Thanks for your comment. Are you fine with this suggestion?
```suggestion
visualiation->visualisation, visualization,
visualsation->visualisation,
visualsation->visualisation,
visulisation->visualisation,
visulisations->visualisations,
visulization->visualization,
visulizations->visualizations,
```

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:35808]: @tkoyama010 single results do not need a trailing `,`

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@peternewman I have pushed the requested changes. Can you have a look and approve?

**Review by skangas — APPROVED** [review]

**Review by luzpaz — APPROVED** [review]

**Comment by tkoyama010** [comment]
Thanks for the reviews!

**Pull request merged**